File: src/Lesson5/java/Ex5.java
package Lesson5.java;

import java.util.Arrays;

public class Ex5 {
    public static void main(String[] args) {

        int[] arr = {1, 2, 3, 4, 5};

        System.out.println("Original array: " + Arrays.toString(arr));

        int temp = arr[0];

        arr[0] = arr[arr.length - 1];

        arr[arr.length - 1] = temp;

        System.out.println("Modified array: " + Arrays.toString(arr));
    }

}

File: src/Lesson5/forkotlin/Ex5.kt
package Lesson5.forkotlin

fun main() {

    val arr = intArrayOf(5, 2, 9, 1, -7, 40)

    println("Original array: ${arr.joinToString()}")

    val temp = arr[0]

    arr[0] = arr[arr.size - 1]

    arr[arr.size - 1] = temp

    println("Modified array: ${arr.joinToString()}")
}